vaadin-crud-flow-integration-tests/src/main/java/com/vaadin/flow/component/crud/examples/ClickRowToEditView.java

```diff
@@ -0,0 +1,54 @@
+package com.vaadin.flow.component.crud.examples;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.crud.Crud;
+import com.vaadin.flow.component.crud.CrudEditorPosition;
+import com.vaadin.flow.component.crud.CrudGrid;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.page.BodySize;
+import com.vaadin.flow.router.Route;
+import com.vaadin.flow.theme.Theme;
+import com.vaadin.flow.theme.lumo.Lumo;
+
+@Route
+@Theme(Lumo.class)
+@BodySize(height = "100vh", width = "100vw")
+public class ClickRowToEditView extends VerticalLayout {
+    public static String SETCLICKTOEDIT_BUTTON_ID =  "setClickToEdit";
+    public static String CLEARCLICKTOEDIT_BUTTON_ID =  "clearClickToEdit";
+
+
+    public ClickRowToEditView() {
+        final CrudGrid<Person> grid = new CrudGrid<>(Person.class, false);
+        final Crud<Person> crud = new Crud<>(Person.class, grid,
+                Helper.createPersonEditor());
+
+        PersonCrudDataProvider dataProvider = new PersonCrudDataProvider();
+
+        grid.setDataProvider(dataProvider);
+        crud.addSaveListener(e -> dataProvider.persist(e.getItem()));
+        crud.addDeleteListener(e -> dataProvider.delete(e.getItem()));
+
+        crud.setEditorPosition(CrudEditorPosition.ASIDE);
+
+        final Button setClickToEdit = new Button("Set Click Row to Edit",
+                event -> crud.setClickRowToEdit(true));
+        setClickToEdit.setId(SETCLICKTOEDIT_BUTTON_ID);
+
+        final Button clearClickToEdit = new Button("Clear Click Row to Edit",
+                event -> crud.setClickRowToEdit(false));
+        clearClickToEdit.setId(CLEARCLICKTOEDIT_BUTTON_ID);
+
+        Label editorPosition = new Label();
+        editorPosition.setId("editorPositionLabel");
```

Comment:
Why do we need it on this page as well as the button below?

Reply:
I copied from another page, but forgot to remove stuff that were not in use.